front/app/utils/dateUtils.test.ts

```diff
@@ -3,13 +3,120 @@ import {
   getIsoDateUtc,
   getIsoDate,
   getIsoDateForToday,
+  timeAgo,
 } from './dateUtils';
 import moment from 'moment';
 import 'moment-timezone';
 
 // test date is 1AM June 15 2020 UTC time (Z)
 const testDateStr = '2020-06-15T01:00:00Z';
 
+// timeAgo should report the elapsed time since a given date correctly
+describe('timeAgo is reported correctly', () => {
+  it('should accurately return seconds passed since a date', () => {
+    let date = new Date();
+    date.setSeconds(date.getSeconds() - 1);
+    let timeAgoResponse = timeAgo(date.valueOf(), 'en') || '';
+    expect(timeAgoResponse).toEqual('1 second ago');
+
+    date = new Date();
+    date.setSeconds(date.getSeconds() - 2);
+    timeAgoResponse = timeAgo(date.valueOf(), 'en') || '';
+    expect(timeAgoResponse).toEqual('2 seconds ago');
+  });
+
+  it('should accurately return minutes passed since a date', () => {
+    let date = new Date();
+    date.setMinutes(date.getMinutes() - 1);
+    date.setSeconds(date.getSeconds() - 1);
```

Comment:
Why the seconds here? :)

Reply:
Ah true - I can use <= in the function instead. I was thinking that if it's even 1 millisecond over the limit (e.g. 60 seconds and 1 millisecond) it will show "1 minute ago" anyways, but using <= simplifies it. Thanks for catching it :)